.github/workflows/pre-commit.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Setup Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v2
```

Comment:
@mariehbourget so it seems... the problem you mentioned was widespread across PRs.